Jenkinsfile

```diff
@@ -28,9 +28,8 @@ node('docker') {
         try {
             stage("Test") {
                 sh """docker run --rm --name ${dockerTestRunner} \\
-                                 -w /go/src/github.com/cyverse-de/${service.repo} \\
-                                 --entrypoint 'gb' \\
-                                 ${dockerRepo} test"""
+                                 --entrypoint 'go' \\
+                                 ${dockerRepo} test /go/src/github.com/cyverse-de/${service.repo}/..."""
```

Comment:
Just a "could be nice", but we could make this generate the nice junit reports in jenkins. To do that, set the entrypoint to `sh` and the command to `-c "go test -v <path> | tee /dev/stderr | go-junit-report"` and tack on a ` > test-results.xml`, then add `junit 'test-results.xml'` and some cleanup below that. Look at, for example, the saved-searches Jenkinsfile, if you're interested (particularly the cleanup details, which are weird to relay in a comment). `go-junit-report` is installed in the golang base image, so that doesn't need to be explicitly added in or anything.